lib/trln_argon/controller_override.rb

```diff
@@ -90,10 +90,6 @@ module ControllerOverride
                                label: TrlnArgon::Fields::AVAILABLE_FACET.label,
                                limit: true,
                                collapse: false
-        config.add_facet_field TrlnArgon::Fields::INSTITUTION_FACET.to_s,
```

Comment:
Why did you remove this facet?

Reply:
It's not a facet we plan to use in production and it kept coming up in meetings that it was confusing so I removed it.